### DISCUSSION TITLE
Publish/Update button still grayed out

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#866] [General]
I am unable to update or publish new meetings as the button grays out as soon as I enter any data in the zoom input or location inputs. We are using Google maps. I am starting a new discussion as requested as the original thread was answered but that solution is not working for me.

**Comment by tim-rohrer** [comment]
I apologize that you're experiencing this issue still. We'll try hard to get it fixed quickly.

Let me know which method of geocoding your site is configured for (Legacy or Beta). You can determine this under Meetings-->Import & Settings-->Settings. 

Open a browser debugging console and then try entering the location address again. When you tab off that field, you'll see "Geocoded" show up in your console. Expand that, then screen cap it, and post it here please.

**Reply by district6wc** [reply]
We have the user interface set to Legacy UI, it looks like this has been corrected. I am seeing the following for Geocoded

Geocoded:  {formatted_address: '513 W Front St, Burlington, NC 27215, USA', latitude: 36.0978765, longitude: -79.44212739999999, approximate: 'no', city: 'Burlington', …}
admin.min.js?ver=3.14.5:1 createMap() locations {0: {…}}
admin.min.js?ver=3.14.5:1 setMapMarkers() location {formatted_address: '513 W Front St, Burlington, NC 27215, USA', latitude: 36.0978765, longitude: -79.44212739999999, approximate: 'no', city: 'Burlington', …}
admin.min.js?ver=3.14.5:1 setMapMarkers() marker 

I do see a lot of blocked by client errors also, not sure if that is something to be concerned about
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.restproxy.ws_supported.chrome:1|c%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.restproxy.ws_supported.chrome:1|c%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.restproxy.nocookie.chrome:1|c%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.restproxy.nocookie.chrome:1|c%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
?v=2.0:16 WebSocket connection to 'wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data' failed: 
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:108|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:108|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onerror @ ?v=2.0:16
error (async)
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:109|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:109|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onclose @ ?v=2.0:16
DevTools failed to load source map: Could not load content for chrome-extension://fheoggkfdfchfphceeifdbepaooicaho/sourceMap/chrome/scripts/iframe_form_check.map: System error: net::ERR_BLOCKED_BY_CLIENT
?v=2.0:16 WebSocket connection to 'wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data' failed: 
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:117|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:117|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onerror @ ?v=2.0:16
error (async)
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:117|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:117|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onclose @ ?v=2.0:16
?v=2.0:16 WebSocket connection to 'wss://public-api.wordpress.com/pinghub/wpcom/me/newest-note-data' failed: 
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:166|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_err.chrome:166|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onerror @ ?v=2.0:16
error (async)
v @ ?v=2.0:16
D @ ?v=2.0:16
O @ ?v=2.0:16
pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:166|ms%22]}:1          GET https://pixel.wp.com/boom.gif?v=0.9&u=https://public-api.wordpress.com/pinghub&json={%22beacons%22:[%22pinghub.conn_close_code.1006.chrome:166|ms%22]} net::ERR_BLOCKED_BY_CLIENT
Image (async)
b @ ?v=2.0:16
e.onclose @ ?v=2.0:16

**Reply by tim-rohrer** [reply]
Glad to hear the geocoding issue is resolved! 

The other errors do not appear to be related to TSML. If they’re not affecting the functionality of your site, not sure I’d worry about them. But if you want to, you should probably start in the general Wordpress forums.
